### ISSUE TITLE
Clarify the need to add sphinx.ext.intersphinx in documentation

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1325]
It took me about 3 hours to find that answer on stack overflow

http://stackoverflow.com/a/8794210/135079

Please, make it clear that sphinx.ext.intersphinx must be added to extensions list if you want to make a link to other document (not other site, just your own reST file).

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1325
- Originally reported by: [peterdemin](https://bitbucket.org/peterdemin)
- Originally created at: 2013-12-17T12:18:58.862

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-12-17 11:51:14+00:00_

There is: http://sphinx-doc.org/ext/intersphinx.html

"http://sphinx.pocoo.org/" is an old official URL.

**Comment by shimizukawa** [comment]
_From [peterdemin](https://bitbucket.org/peterdemin) on 2013-12-17 12:18:12+00:00_

Well, I guess you didn't get my point. I request documentation update, as I see it, tutorial page: http://sphinx-doc.org/tutorial.html

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-12-17 13:10:44+00:00_

Ok , I'll write it.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-12-17 14:59:38+00:00_

I wrote. Please check https://bitbucket.org/birkenfeld/sphinx/pull-request/197.

**Comment by shimizukawa** [comment]
_From [peterdemin](https://bitbucket.org/peterdemin) on 2013-12-17 17:40:33+00:00_

Thank you, Takayuki, it is nice introduction to intersphinx. I haven't found any misspellings.

But it lacks crucial part about extending `:ref:` scope to whole project instead of single document.

From what I read, I would think, that it can be used only for links to external sources (other sites).

Example from linked StackOverflow question illustrates precisely what I mean:

```
..
    File: some_file.rst

.. _ref-section:

Some Section
------------

Paragraph, foo bar lorem ipsum.
```

```
..
    File: some_other_file.rst

.. seealso:: :ref:`ref-section`
```

That `:ref:` will work only if you enable `sphinx.ext.intersphinx`.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-12-17 23:40:10+00:00_

Updated.

BTW, I think we need a comprehensive reference manual for sphinx including intersphinx. Just now I don't know which roles work with intersphinx correctly...

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 13:54:51+00:00_

update a Intersphinx tutorial section. refs #1325

→ <<cset bcff26c6dacbd2642517e43a80cdfeddd011aa6e>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 13:54:51+00:00_

Added a "Intersphinx" tutorial section. (:file:`doc/tutorial.rst`). Closes #1325

→ <<cset 1367f6f9c62076cf3818c12d4be5cd43597c33fe>>
